lib/elixir/lib/code/formatter.ex

```diff
@@ -1060,6 +1076,7 @@ defmodule Code.Formatter do
       fun
       |> Atom.to_string()
       |> string()
+      |> color(:function, state.inspect_opts)
```

Comment:
Not sure if we should call `function` or `call`?

Reply:
Perhaps we could have `qualified_call` and `non_qualified_call`?

Reply:
I like `:call` 👍

> Perhaps we could have qualified_call and non_qualified_call?

I'm not sure how much we want to specialize these: this would offer some extra flexibility but feels a bit overkill.
It would also makes it a bit harder to configure, currently the list is quite small and simple.
Maybe I'm just to used to have the same color for both in my IDE 🙂

Reply:
So let’s go with :call and ship this. We can always add specializations with fall backs later. :)

Reply:
Sounds good 👍 Done in https://github.com/elixir-lang/elixir/pull/12106/commits/12c42c850ec8192d5bf0089685316c1b9e3abc4e.

Another idea of a possible specialization with fallbacks, if we want to go down this road: `:integer`/`:float` (currently: `:number`).